src/CMakeLists.txt

```diff
@@ -78,3 +78,4 @@ foreach(dim RANGE 1 3)
     unset(TestsLib)
 
 endforeach()
+
```

Comment:
Avoid committing trivial changes, otherwise the reviewers see 121 files changed.

Reply:
fixed